LTHPasscodeViewController/LTHPasscodeViewController.h

```diff
@@ -314,6 +314,13 @@
  */
 - (void)showLockScreenWithAnimation:(BOOL)animated withLogout:(BOOL)hasLogout andLogoutTitle:(NSString*)logoutTitle;
 /**
+ @brief				Used for displaying the lock. The passcode view is added directly on the keyWindow.
+ @param superview   The superview where is to be presented, used to measure the center of the view.
+ @param hasLogout   Set to @c YES for a navBar with a Logout button, set to @c NO for no navBar.
+ @param logoutTitle The title of the Logout button.
+ */
+- (void)showLockScreenIntoSuperview:(UIView *)superview WithAnimation:(BOOL)animated withLogout:(BOOL)hasLogout andLogoutTitle:(NSString*)logoutTitle;
```

Comment:
This would sound better at calling site like (note the `withAnimation` has a lowercase `w`):

```objc
(void)showLockScreenIntoSuperview:(UIView *)superview withAnimation:(BOOL)animated logout:(BOOL)hasLogout andLogoutTitle:(NSString*)logoutTitle;
```